### PR TITLE
Maps

### PR DESCRIPTION
**Pull request by LJ171**

### CODE REVIEW

File: src/app/creation-wizard-location/creation-wizard-location.component.html [hunk 1]
```diff
@@ -1,5 +1,14 @@
-<div class="container-fluid h-100 pl-6rem" fxLayout="column" fxLayoutAlign="center stretch">
-  <button routerLink="/report/images" class="btn btn-primary btn-md btn-circle" fxFlexAlign="end" type="button" >
+<div class="container-fluid vh-100 pl-6rem py-5" fxLayout="column" fxLayoutAlign="center stretch">
+  <input #search class="mb-2 form-control" type="address" placeholder="Enter a Location" autocorrect="off" autocapitalize="off" spellcheck="off"/>
+  <div style="width:100%;height:60%">
+    <agm-map [zoom]="zoom" (mapClick)="onMapClicked($event)" style="width:100%;height:100%" [latitude]="latFocus" [longitude]="lngFocus">
+      <agm-marker [latitude]="reportPersistenceService.reportData.Latitude" [longitude]="reportPersistenceService.reportData.Longitude"></agm-marker>
+    </agm-map>
+    <button (click)="setFocusAndMarkerOnCurrentPosition()" class="position-relative btn btn-secondary btn-md btn-circle" style="z-index:3;bottom:5rem;left:1rem;" type="button">
```
Comment (tillgeissler) [inline src/app/creation-wizard-location/creation-wizard-location.component.html]: Use bootstrap classes

File: src/app/creation-wizard-location/creation-wizard-location.component.ts [hunk 2]
```diff
@@ -10,12 +12,65 @@ import { ReportPersistenceService } from '../service/report-persistence.service'
...
+          this.reportPersistenceService.reportData.Latitude = place.geometry.location.lat();
+          this.reportPersistenceService.reportData.Longitude = place.geometry.location.lng();
+          this.latFocus = place.geometry.location.lat();
+          this.lngFocus = place.geometry.location.lng();
+       });
+     });
+   });
+  }
+
+  setFocusAndMarkerOnCurrentPosition() {
```
Comment (tillgeissler) [inline src/app/creation-wizard-location/creation-wizard-location.component.ts]: Focus doesn't work

File: package.json [hunk 3]
```diff
@@ -27,22 +31,24 @@
     "@fortawesome/free-solid-svg-icons": "^5.11.2",
     "@ng-bootstrap/ng-bootstrap": "^5.1.2",
     "@types/jquery": "^3.3.31",
+    "azure-keyvault": "^3.0.5",
```
Comment (tillgeissler) [inline package.json]: not needed => remove

File: src/app/creation-wizard-location/creation-wizard-location.component.ts (hunk 2 above)
Comment (LJ171) [reply src/app/creation-wizard-location/creation-wizard-location.component.ts]: As discussed earlier in class Chris will fix this in a later pr.

**Pull request merged**